libethashseal/EthashProofOfWork.h

```diff
@@ -47,14 +48,38 @@ struct EthashProofOfWork
 
 	struct WorkPackage
 	{
-		WorkPackage() = default;
+		WorkPackage() {}
 		WorkPackage(BlockHeader const& _bh);
-		void reset() { headerHash = h256(); }
-		operator bool() const { return headerHash != h256(); }
+		WorkPackage(WorkPackage const& _other)
```

Comment:
I tried, but I found [one place](https://github.com/ethereum/cpp-ethereum/blob/90db42676885bb0589e6daa2a0d6a81f0d16ba35/libethereum/GenericFarm.h#L73) where a `WorkPackage` is copied into every miner.
```
		for (auto const& m: m_miners)
			m->setWork(m_work);
```

So I should keep it copyable.